### ISSUE TITLE
Selectpicker doesn't work with Angularjs

### ISSUE THREAD

**Issue by alanloffler** [silviomoreto/bootstrap-select#58]
I'm trying to use the selectpicker with angularjs, but doesn't show properly the use of brackets in the value parameter and the option text.

Did someone else tried to use the selectpicker in angularjs with bootstrap?

**Comment by rajmohanpdy** [comment]
i have been facing this problem. selectpicker not working with ng-modal. Any soluttion ?

**Comment by caseyjhol** [comment]
Check out https://github.com/caseyjhol/ng-bootstrap-select

**Comment by rosorio02** [comment]
Hi there!
I know the post was made long time ago but I hope my solution will be a help for others.

I got another solution to the issue related.

What I did was first load my data from back-end using angularjs and inside the .success after my validation I wrote the following code:
angular.element(document).ready(function () { $('select').selectpicker("destroy"); $('select').selectpicker("render"); });
I tried to do it without angular.element(document).ready(function (){}) but nothing happend. I suppose the plugin's methods only works inside that function.

Happy Coding!!
